tutorials/animation/2d_skeletons.rst

```diff
@@ -1,240 +1,376 @@
-built-in.
+Cutout animation in Godot
+~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Why would you want to do skeletal animations directly in Godot? The answer is
-that there are many advantages to it:
+Godot provides tools for working with cutout rigs, and is ideal for the workflow:
 
-* Better integration with the engine, so less hassle importing and editing from
-  an external tool.
-* Ability to control particle systems, shaders, sounds, call scripts, colors,
-  transparency, etc. in animations.
-* The built-in skeletal system in Godot is very efficient and designed for
-  performance.
+-  **The animation system is fully integrated with the engine**: This
+   means animations can control much more than just motion of objects. Textures,
+   sprite sizes, pivots, opacity, color modulation, and more, can all be animated
+   and blended.
+-  **Combine animation styles**: AnimatedSprite2D allows traditional cel animation
+   to be used alongside cutout animation. In cel animation different animation
+   frames use entirely different drawings rather than the same pieces positioned
+   differently. In an otherwise cutout-based animation, cel animation can be used
+   selectively for complex parts such as hands, feet, changing facial expressions,
+   etc.
+-  **Custom Shaped Elements**: Custom shapes can be created with
+   :ref:`Polygon2D <class_Polygon2D>`
+   allowing UV animation, deformations, etc.
+-  **Particle Systems**: A cutout animation rig can be combined with particle
+   systems. This can be useful for magic effects, jetpacks, etc.
+-  **Custom Colliders**: Set colliders and influence areas in different
+   parts of the skeletons, great for bosses and fighting games.
+-  **Animation Tree**: Allows complex combinations and blending between
+   several animations, the same way it works in 3D.
 
-The following tutorial will, then, explain 2D skeletal deformations.
+And much more!
 
-Setup
------
+Making of GBot
+~~~~~~~~~~~~~~
 
-.. seealso::
+For this tutorial, we will use as demo content the pieces of the
+`GBot <https://www.youtube.com/watch?v=S13FrWuBMx4&list=UUckpus81gNin1aV8WSffRKw>`__
+character, created by Andreas Esau.
 
-   Before starting, we recommend you to go through the
-   :ref:`doc_cutout_animation` tutorial to gain a general understanding of
-   animating within Godot.
+.. image:: img/tuto_cutout_walk.gif
 
-For this tutorial, we will be using a single image to construct our character.
-Download it from :download:`gBot_pieces.png <img/gBot_pieces.png>` or save the
-image below.
+Get your assets: :download:`gbot_resources.zip <files/gbot_resources.zip>`.
 
-.. image:: img/gBot_pieces.png
+Setting up the rig
+~~~~~~~~~~~~~~~~~~
 
-It is also advised to download the final character image
-:download:`gBot_complete.png <img/gBot_complete.png>` to have a good reference
-for putting the different pieces together.
+Create an empty Node2D as root of the scene, we will work under it:
```

Comment:
Starting from here, the names for the bone and sprite nodes are different from the defaults, but there are no instructions to change the name of the nodes. How about

> Create an empty Node2D as the root of the scene, and rename it as "GBot". We will work under it: